src/components/social-media-feed.tsx

```diff
@@ -0,0 +1,41 @@
+import { useEffect, useState } from "react";
+import Image from "next/image";
+
+export default function FacebookFeed() {
+  /*const [facebookData, setFacebookData] = useState(null);
+    useEffect(() => {
+      const fetchData = async () => {
+        const data = await fetch("/api/facebook-feed", {
+          method: "GET"
+        }).then(res => res.json());
+        setFacebookData(data);
+      }
+      fetchData();
+    }, []);
+    if (facebookData) {
+      return (<p>{facebookData}</p>);
+    } else {
+      return null;
+    }*/
+  return (
+    <div>
+      <iframe
+        src="https://www.facebook.com/plugins/page.php?href=https%3A%2F%2Fwww.facebook.com%2FStarickServices&tabs=timeline&width=340&height=500&small_header=true&adapt_container_width=true&hide_cover=true&show_facepile=true&appId"
+        width="340"
+        height="500"
+        title="facebook-embed"
+      ></iframe>
+      <div className="flex flex-row items-center justify-center">
+        <a href="https://www.instagram.com/starickservices/">
```

Comment:
It might be better UX if the Instagram text also can be clicked on to go to the Instagram page as well (right now it's just the icon). Maybe also some custom styling around these two elements to make it more obviously a button as well - right now it looks a bit like plain text